spec/lib/exact_target_rest/triggered_send_spec.rb

```diff
@@ -68,7 +68,7 @@
           subscriber_attributes: { "City" => "São Paulo", "Profile ID" => "42" }
         )
 
-      expect(YAML::load(ts.to_yaml)).to be_instance_of(ExactTargetRest::TriggeredSend)
+      expect(YAML::unsafe_load(ts.to_yaml)).to be_instance_of(ExactTargetRest::TriggeredSend)
```

Comment:
Since we updated to ruby 3.1.0 yaml has changed so now it restricts what classes it will load with a conventional load so they added an unsafe_load and since this is only used in the data round-trip tests it should be fine to use it here.